logging/google/cloud/logging/handlers/transports/background_thread.py

```diff
@@ -230,7 +236,7 @@ def _main_thread_terminated(self):
                 file=sys.stderr)
 
     def enqueue(self, record, message, resource=None, labels=None,
-                trace=None, span_id=None):
+                trace=None, span_id=None, include_logger_name=True):
```

Comment:
`include_logger_name` is passed, but in the method we use the instance method, `self._include_logger_name`.

Reply:
Done!